src/main/java/racing/Cars.java

```diff
@@ -6,26 +6,31 @@
 
 public class Cars {
 
-    private final static MoveStrategy randomMove = new RandomMove();
-
     private final List<Car> cars;
 
     public Cars(Integer numberOfCar) {
+        if (isZeroOrLess(numberOfCar)) {
```

Comment:
여기는 아래 코드가 좀 더 직관 적이지 않을까요?
로직이 한눈에 들어오면 굳이 함수로 추출할 필요가 있을까 해요
```suggestion
        if (numberOfCar <= 0) {
```